docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -604,6 +605,14 @@ Outputs a QR-code that represents URLs used to connect the [remote browsers](#re
 testcafe remote my-tests --qr-code
 ```
 
+### --sf, --stop-on-first-fail
+
+Stops an entire test run if any test fails. This allows you not to wait for all the tests included in the test task to finish and allows focusing on the first error.
```

Comment:
> allows you not to wait

I feel like this expression is not quite easy to read

> allows focusing

Are you sure this is correct?

I would say something like

> Stops the entire test run once any test fails. Enable this option if you don't want to wait for all the tests included in the test task to finish and instead want to focus on the first error.

Reply:
I think my phrase is good. My text was checked by Yana Orlova, it is correct.